reference/features/filtering_and_faceted_search.md

```diff
@@ -0,0 +1,318 @@
+        "director": "Jordan Peele",
+        "poster": "https://image.tmdb.org/t/p/w1280/ux2dU1jQ2ACIMShzB3yP93Udpzc.jpg",
+        "overview": "Husband and wife Gabe and Adelaide Wilson take their…",
+        "release_date": 1552521600,
+        "genres": [
+            "Comedy",
+            "Horror",
+            "Thriller"
+        ],
+        "rating": 4
+    },
+    …
+]
+```
+
+If you want to enable filtering using `director`, `release_date`, `genres`, and `rating`, you must add these attributes to the [`filterableAttributes` index setting](/reference/api/filterable_attributes.md).
+
+You can then restrict a search so it only returns movies released after 18 March 1995 with the following filter containing a single condition:
+
+```SQL
+release_date > 795484800
+```
+
+You can use this filter when searching for recent `Avengers` movies:
+
+<CodeSamples id="filtering_guide_1" />
+
+You can also combine multiple conditions. For instance, you can limit your search so it only includes recent movies directed by either `Tim Burton` or `Christopher Nolan`:
+
+```SQL
+release_date > 795484800 AND (director = "Tim Burton" OR director = "Christopher Nolan")
+```
+
+Here, the parentheses are mandatory: without them, the filter would return movies directed by `Tim Burton` and released after 1995 or any film directed by `Christopher Nolan`, without constraints on its release date. This happens because `AND` takes precedence over `OR`.
+
+You can use this filter when searching for `Batman` movies:
+
+<CodeSamples id="filtering_guide_2" />
+
+Note that filtering on string values is case-insensitive.
+
+If you only want well-rated movies that weren't directed by `Tim Burton`, you can use this filter:
+
+```SQL
+rating >= 3 AND (NOT director = "Tim Burton")
+```
+
+Querying on `Planet of the Apes`, the above example results in the following command:
+
+<CodeSamples id="filtering_guide_4" />
+
+## Faceted search
+
+**Faceted search** is a feature provided out-of-the-box by MeiliSearch. Faceting allows classifying search results into categories or **facets**. Facets are essentially a type of filter.
+
+Faceted search provides users with a quick way to narrow down search results by selecting categories relevant to what they are looking for. A faceted navigation system is an **intuitive interface to display and navigate through content**. Facets are used in the UI as filters which users can apply to refine the results in real-time.
+
+This is common in ecommerce sites like Amazon: when users perform a search, they are presented not only with a list of results, but also with a list of facets which you can see on the sidebar in the image below:
+
+![Amazon UI](/faceted-search/facets-amazon.png)
+
+### Filters or facets
+
+Since MeiliSearch v0.21, there is no difference between facets and filters. They function and behave in the exact same way. The question of whether something is a filter or a facet is mostly one pertaining to UX and UI design.
```

Comment:
This need to be reworded as facets are a sub-category of filters

Reply:
Rather than a sub-category, I think we should say that facetting is a way to filter from a user experience point of view. The term sub category doesn't seem very clear to me for a user.

Also, do we want to mention the version number?

Reply:
> Rather than a sub-category, I think we should say that facetting is a way to filter from a user experience point of view. The term sub category doesn't seem very clear to me for a user.

I agree with this and it was probably the biggest challenge in the whole article: as far as I understand it, facets are not a different _type_ of filter, they are a different _use_ of filters.

Reply:
```suggestion
### Filters or facets

In MeiliSearch, facets are a specific use-case of filters. The question of whether something is a filter or a facet is mostly one pertaining to UX and UI design.
```